static/app/components/replays/breadcrumbs/replayTimeline.tsx

```diff
@@ -56,7 +59,7 @@ function ReplayTimeline({size}: Props) {
     <VisiblePanel ref={panelRef} {...mouseTrackingProps}>
       <Stacked
         style={{
-          width: `${size}%`,
+          width: `${timelineSize}%`,
           transform: `translate(${translatePercentage}, 0%)`,
```

Comment:
nit:
I'd call `toPercent` on both of these lines, which leaves the values as numbers for as long as possible:
```suggestion
          width: toPercent(timelineSize),
          transform: `translate(${toPercent(translatePercentage)}, 0%)`,
```